src/DetailsView/components/issues-table.tsx

```diff
@@ -45,29 +43,14 @@ export interface IssuesTableProps {
     userConfigurationStoreData: UserConfigurationStoreData;
 }
 
-export interface IssuesTableState {
-    isExportDialogOpen: boolean;
-    exportDescription: string;
-    exportName: string;
-    exportDataWithPlaceholder: string;
-    exportData: string;
-}
-
-export class IssuesTable extends React.Component<IssuesTableProps, IssuesTableState> {
+export class IssuesTable extends React.Component<IssuesTableProps> {
```

Comment:
It would be nice to convert this component (`IssuesTable` & `DetailsViewCommandBar `) to a `NamedSFC` but I can totally see that as a different PR!

Reply:
that's a good call.